data/sql/updates/pending_db_world/rev_1626780409112630500.sql

```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1626780409112630500');
+
+-- Added movement to the default spawn point he had
+UPDATE `creature` SET `wander_distance` = 20, `MovementType` = 1 WHERE (`id` = 14343) AND (`guid` = 51897);
+
+-- Added 2 more spawn points and movement on them
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(301303, 14343, 1, 0, 0, 1, 1, 0, 0, 6091.22, -1485.77, 438.13, 0.18, 9900, 20, 0, 1, 0, 1, 0, 0, 0, '', 0),
+(301304, 14343, 1, 0, 0, 1, 1, 0, 0, 6307.9, -1637.18, 474.611, 0, 9900, 20, 0, 1, 0, 1, 0, 0, 0, '', 0);
```

Comment:
Please use an ``UPDATE`` here too, precisely because of the transparency and compactness and because the code can then be repeated without an error message

Reply:
The problem i see is that the ids i used (301303 and 301304) were not in the database to start with.
I dont know if its possible to use the update without having the primary keys added first, and i need them for the where clause.

Reply:
> The problem i see is that the ids i used (301303 and 301304) were not in the database to start with.
> I dont know if its possible to use the update without having the primary keys added first, and i need them for the where clause.

Ok, in that case you need to delete the entries first and the ``INSERT`` is ok here

Reply:
No problem at all!
I added the delete clause before the inserts as you said.